ddtrace/profiling/collector/stack.pyx

```diff
@@ -12,6 +15,23 @@ from ddtrace.profiling import event
 from ddtrace.profiling.collector import _traceback
 from ddtrace.utils import formats
 from ddtrace.vendor import attr
+from ddtrace.vendor import six
+
+
+_LOG = logging.getLogger(__name__)
+
+
+if "gevent" in sys.modules:
```

Comment:
should we be checking `gevent.monkey.is_module_patched("threading")`? I'm not sure if there are people that are importing gevent and not actually using it, but maybe? Would be good to be precise here 😄

Reply:
Not really a big deal, since patched or not you'll get the original function, which is what we want.